stable/kong/values.yaml

```diff
@@ -51,13 +51,21 @@ proxy:
   annotations: {}
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
-  # HTTPS traffic on the proxy port
-  useTLS: true
-  servicePort: 8443
-  containerPort: 8443
+  # HTTP plain-text traffic
+  http:
+    enabled: true
+    servicePort: 80
+    containerPort: 8000
+    # nodePort: 32080
+
+  tls:
+    enabled: true
+    servicePort: 443
+    containerPort: 8443
+    # nodePort: 32443
+
   type: NodePort
-  # Set a nodePort which is available
```

Comment:
I think this comment was valuable and advises user on taking an action. Let's duplicate it to the two locations where the user can specify nodePort now.